documentation/content/codingstyle.rst

```diff
@@ -209,6 +209,16 @@ ideally, we want it in the same file. For example, you should find details on
 using the `build` option in the mk.sh, and for `configm` in the file
 config_manager.sh.
 
+Handling Signals
+----------------
+When we execute some specific command, it is natural that we set global
+variables or create temporary files. However, all commands should expect
+receiving signals and should deal with each of them properly. If you implement a
+new feature, take some time to check if it pollutes the environment. If it does,
+handle it's depollution upon receiving SIGINT or SIGTERM. Convenience functions
+for this purpose (setting and resetting handlers for arbitrary signals) are
+implemented in `src/signal_manager`.
```

Comment:
how about:
" It is natural for commands to set global variables or to create temporary files during their execution. However, all commands should expect to receive signals and be able to properly handle them. If you implement a
new feature, take some time to check if it pollutes the environment. If it does,
make sure to handle it's de-pollution upon receiving a SIGINT or a SIGTERM, an interrupted command should always leave the environment in the same state as it was prior to its invocation. (...)"